### PR TITLE
📝  Add more information to code repos page

### PR DESCRIPTION
**Pull request by robertdeniszczyc2**
Adds a brief description to the code repos, adds a GH PR filter section to the code repos page.

### CODE REVIEW

File: documentation/code-repos.md [hunk 1]
```diff
@@ -10,70 +10,103 @@ Almost all HOCS code is open-source and publicly available on GitHub.
 
 ### Core components
 [hocs-audit](https://github.com/UKHomeOffice/hocs-audit)
+Receives audit event messages from an SQS queue for persistent storage.
 
 [hocs-case-creator](https://github.com/UKHomeOffice/hocs-case-creator)
+Reads a message containing a JSON payload from an AWS SQS Queue, and creates a cases with associateed data.
```
Comment (LiamMacP) [inline documentation/code-repos.md:16]:
```suggestion
Reads a message containing a JSON payload from an AWS SQS Queue, and creates a cases with associated data.
```

File: documentation/code-repos.md [hunk 2]
```diff
@@ -10,70 +10,100 @@ Almost all HOCS code is open-source and publicly available on GitHub.
...
+Handles interactions with reference data specific to cases.
 
 [hocs-data 🔒️ *(to be deprecated)*](https://github.com/UKHomeOffice/hocs-data)
+The CS schema.
 
 [hocs-data-wcs 🔒️ *(to be deprecated)*](https://github.com/UKHomeOffice/hocs-data-wcs)
+The WCS schema.
 
 [hocs-docs](https://github.com/UKHomeOffice/hocs-docs)
+Handles interactions with documents stored in S3.
```
Comment (LiamMacP) [inline documentation/code-repos.md:28]: Worth mentioning that this also handles the interactions with conversion and virus checking?

File: documentation/code-repos.md [hunk 3]
```diff
@@ -10,70 +10,100 @@ Almost all HOCS code is open-source and publicly available on GitHub.
...
+Accepts any document as a parameter and returns a `.pdf` document.
 
 [hocs-info-service](https://github.com/UKHomeOffice/hocs-info-service)
+Handles various reference data interactions.
 
 [hocs-notify](https://github.com/UKHomeOffice/hocs-notify)
+Sends emails to users based on system events.
 
 [hocs-search](https://github.com/UKHomeOffice/hocs-search)
+Serves the search functionality through the use of `elasticsearch`.
```
Comment (LiamMacP) [inline documentation/code-repos.md:40]:
```suggestion
Serves the search functionality through the use of `elasticsearch`/`opensearch`.
```

File: documentation/code-repos.md [hunk 4]
```diff
@@ -10,70 +10,100 @@ Almost all HOCS code is open-source and publicly available on GitHub.
...
 ### UI components
 [hocs-frontend](https://github.com/UKHomeOffice/hocs-frontend)
+[React](https://reactjs.org/) application and [NodeJS](https://nodejs.org/en/) Server for server rendered dynamic forms.
 
 [hocs-management-ui](https://github.com/UKHomeOffice/hocs-management-ui)
+[React](https://reactjs.org/) application for performing management utility functions.
 
 ### Utility components
 [hocs-casework-search-indexer](https://github.com/UKHomeOffice/hocs-casework-search-indexer)
+Reindexs all data in hocs-casework to the hocs-search `elasticsearch index.
```
Comment (LiamMacP) [inline documentation/code-repos.md:57]:
```suggestion
Reindexs all data in hocs-casework to the hocs-search `elasticsearch` index.
```

File: documentation/code-repos.md [hunk 5]
```diff
@@ -10,70 +10,100 @@ Almost all HOCS code is open-source and publicly available on GitHub.
...
 
 [hocs-toolbox](https://github.com/UKHomeOffice/hocs-toolbox)
+Tools to interact with an environment's databases and AWS resources.
```
Comment (LiamMacP) [inline documentation/code-repos.md:60]:
```suggestion
Tools to interact with an environment's databases.
```

File: documentation/code-repos.md [hunk 6]
```diff
@@ -10,70 +10,100 @@ Almost all HOCS code is open-source and publicly available on GitHub.
...
 
 [hocs-queue-tool](https://github.com/UKHomeOffice/hocs-queue-tool)
+A utility to investigate and manage Dead Letter Queues.
```
Comment (LiamMacP) [inline documentation/code-repos.md:63]: This actually interacts with all aspects of queues, from attributes, to purging both SQS and DLQ.

**Comment by robertdeniszczyc2** [comment]
Converted to Draft while I remove the change to the `ci` directory.

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**